packages/web-components/fast-components/src/radio-group/radio-group.ts

```diff
@@ -0,0 +1,174 @@
+
+export class RadioGroup extends FASTElement {
+    @attr({ attribute: "readonly", mode: "boolean" })
+    public readOnly: boolean; // Map to proxy element
+    private readOnlyChanged(): void {
+        const filteredRadios = this.getFilteredRadioButtons();
+        if (filteredRadios !== undefined) {
+            filteredRadios.forEach((radio: HTMLElement) => {
+                if (this.disabled) {
+                    radio.setAttribute("readonly", "");
+                } else {
+                    radio.removeAttribute("readonly");
+                }
+            });
+        }
+    }
+
+    @attr({ attribute: "disabled", mode: "boolean" })
+    public disabled: boolean; // Map to proxy element
+    private disabledChanged(): void {
+        const filteredRadios = this.getFilteredRadioButtons();
+        if (filteredRadios !== undefined) {
+            filteredRadios.forEach((radio: HTMLElement) => {
+                if (this.disabled) {
+                    radio.setAttribute("disabled", "");
+                } else {
+                    radio.removeAttribute("disabled");
+                }
+            });
+        }
+    }
+
+    @attr
+    public name: string; // Map to proxy element
+    protected nameChanged(): void {
+        this.getFilteredRadioButtons().forEach((radio: HTMLElement) => {
+            radio.setAttribute("name", this.name);
+        });
+    }
+
+    @attr({ attribute: "selected-value" })
+    public selectedValue: string;
+    private selectedValueChanged(): void {
+        this.$emit("change");
+    }
+
+    @observable slottedRadioButtons: Node[];
+    private selectedRadio: FASTRadio | HTMLElement;
+
+    constructor() {
+        super();
+        this.addEventListener("keydown", this.keydownHandler);
+    }
+
+    public connectedCallback(): void {
+        super.connectedCallback();
+
+        const radioButtons: HTMLElement[] = this.getFilteredRadioButtons();
+        radioButtons.forEach((radio: HTMLElement) => {
+            radio.addEventListener("change", this.handleRadioChange);
+            if (this.name !== undefined) {
+                radio.setAttribute("name", this.name);
+            }
+
+            if (this.disabled) {
+                radio.setAttribute("disabled", "");
+            }
+
+            if (this.readOnly) {
+                radio.setAttribute("readonly", "");
+            }
+
+            if (
+                this.selectedValue &&
+                this.selectedValue === radio.getAttribute("value")
+            ) {
+                this.selectedRadio = radio;
+                radio.setAttribute("checked", "");
+                radio.setAttribute("tabindex", "0");
+            } else {
+                radio.setAttribute("tabindex", "-1");
+            }
+        });
+
+        if (this.selectedValue === undefined) {
+            radioButtons[0].setAttribute("tabindex", "0");
+        }
+    }
+
+    private getFilteredRadioButtons = (): any[] => {
```

Comment:
I promise to make this easier in the future through the core library. Thanks for working it out in the meantime and for your patience!